### PR TITLE
Quit with Spotify

### PR DESCRIPTION
**Pull request by lbordino**

### CODE REVIEW

**Comment by cfvescovo** [comment]
Well done @lbordino!

**Comment by cfvescovo** [comment]
It works flawlessly

**Comment by cfvescovo** [comment]
@simonmeusel Could you review and merge this PR?

**Comment by qmnl** [comment]
Thank you for the contribution!

I'm now back and have access to computer again so I can review this.

I like the idea of closing Spotify with MuteSpotifyAds. What do you think of making it a separate quit button instead of another options? Because I think the option just adds another step.

So that there is

```
 - Close with Spotify
 - Quit
```

File: MuteSpotifyAds/SpotifyManager.swift [hunk 1]
```diff
@@ -108,6 +109,7 @@ class SpotifyManager: NSObject {
         process.launch()
         process.waitUntilExit()
     }
+
```
Comment (qmnl) [inline MuteSpotifyAds/SpotifyManager.swift]: Too much spacing between functions

File: MuteSpotifyAds/SpotifyManager.swift [hunk 2]
```diff
@@ -232,6 +234,13 @@ class SpotifyManager: NSObject {
         })
     }
     
+    func closeSpotify() {
+
```
Comment (qmnl) [inline MuteSpotifyAds/SpotifyManager.swift]: Please remove the blank lines in this function.

File: MuteSpotifyAds/AppDelegate.swift [hunk 3]
```diff
@@ -24,15 +25,21 @@ class AppDelegate: NSObject, NSApplicationDelegate, NSUserNotificationCenterDele
...
     var spotifyManager: SpotifyManager?
     
     @IBAction func quit(_ sender: Any) {
+        if spotifyManager!.quitSpotify == true {
+            spotifyManager?.closeSpotify()
+        }
+        
         NSApplication.shared.terminate(self)
     }
+
```
Comment (qmnl) [inline MuteSpotifyAds/AppDelegate.swift]: Please remove this blank line

**Comment by cfvescovo** [comment]
@simonmeusel Yes, I think that this would be better.

**Comment by qmnl** [comment]
Thank you!

**Comment by qmnl** [comment]
Released with `v1.11.0`: https://github.com/simonmeusel/MuteSpotifyAds/releases

**Pull request merged**